docs/UserGuide.md

```diff
@@ -171,6 +173,23 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
+### Editing a record : `redit`
+
+Edits an existing record in the record list.
+
+Format: `redit INDEX [d/DATE] [r/DATA] [m/MEDICATION]…​`
+
+* Edits the record at the specified `INDEX`. The index refers to the index number shown in the displayed record list. The index **must be a positive integer** 1, 2, 3, …​
+* At least one of the optional fields must be provided.
+* Existing values will be updated to the input values.
+* When editing medication, the existing medication in the record will be removed i.e adding of tags is not cumulative.
+* You can remove all the specific record’s medications by typing `t/` without
+  specifying any tags after it.
+
+Examples:
+*  `editr 1 d/12-12-2012 1200` Edits the date of the 1st record to be `12-12-2012 1200`.
+*  `editr 2 r/Fever m/` Edits the data of the 2nd record to be `Fever` and clears all existing medication.
```

Comment:
Should this be `redit` instead of `editr`?

Reply:
Yup, changing that